### PR TITLE
Add new md file

### PR DESCRIPTION
**Pull request by sohambhattacharjee**
Ready for review. @santhimuthusamy  please review and approve.

### CODE REVIEW

**Comment by sohambhattacharjee** [comment]
@govind999 are you getting notification for this?

File: sb-kolkata.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Soham Bhattacharjee
+
+### Things to do
+- [ ] Code
```
Comment (santhimuthusamy) [inline sb-kolkata.md]: what code?

**Review by santhimuthusamy — CHANGES_REQUESTED** [review]

File: sb-kolkata.md (hunk 1 above)
Comment (sohambhattacharjee) [reply sb-kolkata.md]: This is markdown.

**Comment by sohambhattacharjee** [comment]
@santhimuthusamy , I've changed the code, please approve.

**Review by santhimuthusamy — APPROVED** [review]
looks good

**Review by santhimuthusamy — APPROVED** [review]

**Pull request merged**